.github/workflows/build-container.yaml

```
@@ -0,0 +1,130 @@
+            type=ref,event=pr
+            type=semver,pattern=v{{version}}
+            type=semver,pattern=v{{major}}.{{minor}}
+            type=semver,pattern=v{{major}}.{{minor}}.{{patch}}
+
+      - name: Buildah Build
+        id: build-image
+        uses: redhat-actions/buildah-build@v2
+        with:
+          containerfiles: ${{ matrix.app }}/Containerfile
+          context: ${{ matrix.app }}
+          tags: ${{ steps.meta.outputs.tags || steps.meta_pr.outputs.tags }}
```

actionlint (1.7.7)

property "meta_pr" is not defined in object type {meta: {conclusion: string; outcome: string; outputs: {annotations: string; bake-file: string; bake-file-annotations: string; bake-file-labels: string; bake-file-tags: string; json: string; labels: string; tags: string; version: string}}}

(expression)